lib/twingly/public_suffix_list.rb

```diff
@@ -1,3 +1,4 @@
+require "addressable/idna"
```

Comment:
We do `Addressable::IDNA.to_ascii` here (from before, not added in this PR – but there were no specs then, so it made sense to include it in the commit creating the spec file)